pyroclast/luigi/pyro_tasks.py

```diff
@@ -0,0 +1,31 @@
+import luigi
+import functools
+import itertools
+import json
+import random
+
+from pyroclast.luigi.train_task import TrainTask
+
+
+class TopTask(luigi.Task):
```

Comment:
Huh? What does this do?

Reply:
Also, it looks like it's just running other tasks, so it should extend `luigi.WrapperTask`.  Further, we don't actually need any top level task, unless you want your pretty trees in the visualizer.

Reply:
I'll look into that, ty